### DISCUSSION TITLE
Issue after initializing bash

### DISCUSSION THREAD

**Discussion by JishnuJBhatt** [biostars/biostar-handbook-issues#259] [General]
I have been attempting to re-install windows subsystem for linux. I followed all the instructions in the biostar handbook and regardless of whether I do the Quick Install or the Step-by-Step, the installation fails. After doing it Step-by-Step, I found that the issue occurs after the "Initialize bash" step. When I restart the linux terminal after the conda installation, I see an error in my terminal show up:

```
-bash: eval: line 159: syntax error near unexpected token `('
-bash: eval: line 159: `export CONDA_PROMPT_MODIFIER='(base) ''
```

I have attempted trying the most updated linux subsystem for windows (22.04.5) and the default in the windows store (22.04.1). I have tried updating my wsl1 images to wsl2 via powershell. I purchased the newest version of biostar handbook (Feb2023), and ran the step-by-step code. But all results in the above error.

Once I see that error show up in a freshly opened up terminal, mamba installation always fails with the following error:

```
jb@JB-PC ~
$ conda install mamba -n base -c conda-forge -y
Collecting package metadata (current_repodata.json): failed

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/conda-forge/linux-64/current_repodata.json>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
'https//conda.anaconda.org/conda-forge/linux-64'
```
I am certain nothing is wrong with my internet connection on the pennstate campus. I also tried using my home internet connection with the same issue.
Can you help me with resolving this?

**Comment by ialbert** [comment]
I would completely uninstall WSL then redo the installation. You can't reliably upgrade WSL1 to WSL2

the errors you get is most curious since conda is not a bioinformatics tools and is used by very large number of people well beyond bioinformaticians. 

Both errors are strictly conda specific. It is quite possible that Penn State network interferes with conda under some circumstances, though for me it usually works.

Penn State and other large organizations run their VPN services, sometimes your computer is set up to work through a proxy (especially if it is a work computer).

I would try again, all of these sound like technical IT problems that are not related to bioinformatics.
